### PR TITLE
[#932] Margin fixed (connects #932)

### PR DESCRIPTION
**Pull request by Pr0chin**

### CODE REVIEW

**Comment by Pr0chin** [comment]
capture

**Comment by Pr0chin** [comment]
capture1

**Comment by Pr0chin** [comment]
capture3

**Comment by Pr0chin** [comment]
capture4

**Comment by Pr0chin** [comment]
capture

**Comment by lmmrssa** [comment]
@Pr0chin why does test fails after the merge of dev. And why is it not rebased? Please make the test success.

**Comment by Pr0chin** [comment]
@lmmrssa : dai m on it ..one test has been passed. just waiting for another.

File: app/MyApp/app/Home.css [hunk 1]
```diff
@@ -1024,6 +1022,8 @@ form > fieldset > ul > li.bbf-field.field-selectLanguage {
 #accept div input[name=accept]{
     margin-top: -45px;
 }
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (dogi) [inline app/MyApp/app/Home.css:1026]: very strange

File: app/MyApp/app/Home-Urdu.css [hunk 2]
```diff
@@ -1225,6 +1223,8 @@ form > fieldset > ul > li.bbf-field.field-selectLanguage {
 }
 #accept div input[name=accept]{
     margin-top: -45px;
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (dogi) [inline app/MyApp/app/Home-Urdu.css:1227]: very strange

File: app/CHANGELOG.html [hunk 3]
```diff
@@ -1,4 +1,13 @@
 <html><body><pre>
+ Bell Apps 0.13.19
+ --------------------
+ - [#961] community display in chrome fixed (connects #961) (#1027)
+ - [#1022] mock data for course (connects #1022) (#1023)
+ - [#1028] travis test for delete report fixed (#1029)
+ - [#870] fix download survey button (#1024)
+ - [#829] normalize codecept test names (connects #829) (#1014)
+ - [#991] fix selection of resources in courses (connects #991) (#992) 
+
```
Comment (dogi) [inline app/CHANGELOG.html:10]: ?

**Comment by lmmrssa** [comment]
@Pr0chin  created new PR for it #1035

**Pull request closed without merging**